docs/geedocs/answer/4412421.html

```diff
@@ -252,7 +252,7 @@
     <p>Each resource in a project is known as a <i>layer</i>. Each layer can have one set of display rules. Each display rule includes feature and label formatting that you specify and one or more filters for the selected layer. The filters for each display rule determine which data in the layer to apply that formatting to. For example, in this exercise, you work with road data that includes major and minor highways. You might want the major highways to appear as blue lines and the minor highways as yellow lines. To accomplish that, you create two display rules for the layer--one for major highways and one for minor highways. The filter(s) for each rule determine which data are affected by that rule’s formatting specifications. </p>
     <p>In this exercise, you create and modify a number of display rules for the CAHighways layer, so Google Earth EC displays the road information at the desired display levels with appropriate labels and coloring to distinguish between major freeways and other roads. </p>
     <p>A key part of knowing how to configure display rules for vector data requires familiarity with the data fields in the source data. For the CAHighways layer of your vector project in this tutorial, you can refer to the description of the data fields at this web site: </p>
-    <code>http://www.nationalatlas.gov/mld/roadtrl.html </code>
+    <code>https://nationalmap.gov/small_scale/mld/roadtrl.html</code>
```

Comment:
This is more of a question than a comment.  The linked website doesn't describe the data fields directly, though you can get that information if you download the file and look at the .txt file inside the archive.  I think the easiest way to fix it would be to make the sentence more generic: "You can find information about the CAHighways layer of your vector project at this web site:"

I'm open to other solutions, though.